js/index.js

```diff
@@ -1 +1,77 @@
 // Your code goes here
+
+//==========  entire page/window event listeners
+const home = document.querySelector('.home');
+home.addEventListener('wheel', function(e) {
+    console.log("Wheeeee! We're scrolling, now.");
+})
+
+
+window.addEventListener('load', function(e) {
+    TweenMax.from('.logo-heading', 5, {
+        x: 650,
+        scale: 0.5,
+        opacity:0.1,
+        ease: Bounce.easeOut,
+        delay: 0.5,
+    });
+})
+
+//========== Header event listeners
+const pageHeader = document.querySelector('.main-navigation');
+pageHeader.addEventListener('mouseenter', function(e) {
+    pageHeader.style.backgroundColor = '#17A2B8';
+})
+
+
+const links = document.querySelectorAll('.nav-link');
+for (let i = 0; i < links.length; i++) {
+    links[i].addEventListener('click', function(e) {
+        e.preventDefault;
+        e.stopPropagation();
+        links[i].style.color = '#FFFFFF';
+        
+    })
+}
+
+
+//========== Main content event listeners
+const busPic = document.querySelector('.bus-pic');
+busPic.addEventListener('dragend', function(e) {
+    busPic.src = 'img/photo-1464851707681-f9d5fdaccea8.jpeg'
```

Comment:
Imagine my surprise when I pulled on the fun bus and got a new picture! :laughing: Nice event!